GameServer/Logic/GameObjects/Spell.cs

```diff
@@ -379,7 +423,7 @@ public float getEffectValue(int effectNo)
             return effects[effectNo][Level];
         }
 
-        public void addProjectile(string nameMissile, float toX, float toY)
+        public void addProjectile(string nameMissile, float toX, float toY, bool isServerProjectile = false)
```

Comment:
I think `isServerOnly` would be a better name for the new boolean, as it implies exclusivity whereas the current doesn't necessarily imply that.